### PR TITLE
Ml 10 develop a baseline model

### PR DESCRIPTION
**Pull request by promatty**

### CODE REVIEW

**Review by justin-phxm — CHANGES_REQUESTED** [review]
- Your Analysis looks good. I would throw it into ChatGPT to clean it up though.
There's a few things we can do to improve it. 

1. Clean the data
2. use different algorithms
3. consider using a meta-model
4. train on different subsets of the training data (cross validation)
5. ask chatGPT lmao

**Review by justin-phxm — CHANGES_REQUESTED** [review]
Even after reevaulation of performance, there should be no negative minimum distance:
image

Great summary on the various regression algorithms. My prediction is that Random Forest is going to perform the best.
image

Can you also include an interpretation of the data at the end... What does this data mean in the real world? How can we make data driven decisions from this? For example, maybe an explanation of how this model can predict the distance of a given set of data with +-10m accuracy.
image


You can honestly remove this one section as it is not very relevant. Instead, add a section at the top about how we want a regression algorithm instead of a classification model.
image


You can also turn this into a function as it is duplicated multiple times.
image

I believe it is standard practice to include all the imports at the beginning of the file. Please move this to the top.
image


Look into Hyperparameter tuning... We won't implement this in the current ticket but we will definitely use it in the future to evaluate our models. It's not nearly as important in LinearRegression models but will be huge when it comes to decisionTrees/Forests.

**Review by justin-phxm — APPROVED** [review]
I was hoping that removing the unrealistic negative values would decrease MAE but it's fine. This can happen sometimes
